src/org/ods/orchestration/usecase/JiraUseCase.groovy

```diff
@@ -299,6 +299,11 @@ class JiraUseCase {
         }
     }
 
+    private boolean mustRun(testIssue) {
+        return !project.promotingToProd() ||
+            testIssue.testType?.equalsIgnoreCase(Project.TestType.INSTALLATION)
```

Comment:
or acceptance (installation MUST always run ...)

Reply:
Yes, I reversed the test. Now it says that tests of type installation MUST run. All others are ignored when deploying into prod.